rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```
@@ -106,6 +106,91 @@
         }
       }
     },
+    "type": {
+      "title": "Item type",
+      "type": "string",
+      "enum": [
+        "standard",
+        "issue"
+      ],
+      "default": "standard"
+    },
+    "issue": {
+      "title": "Issue information",
+      "type": "object",
+      "required": [
+        "display_text",
+        "received_date",
+        "expected_date",
+        "regular",
+        "status"
+      ],
+      "propertiesOrder": [
+        "display_text",
+        "received_date",
+        "expected_date",
+        "status"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "status": {
+          "type": "string",
+          "title": "Issue status",
+          "default": "received",
+          "enum": [
+            "received",
+            "claimed"
+          ]
+        },
+        "display_text": {
+          "title": "The displayed text for the predicted issue",
+          "type": "string",
+          "minLength": 4
+        },
+        "received_date": {
+          "type": "string",
+          "format": "date",
+          "title": "The received date for the issue",
+          "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+          "validationMessage": "Should be in the following format: 2022-12-31 (YYYY-MM-DD).",
+          "form": {
+            "type": "datepicker",
+            "wrappers": [
+              "form-field"
+            ],
+            "validation": {
+              "messages": {
+                "pattern": "Should be in the ISO 8601, YYYY-MM-DD."
+              }
+            }
+          }
+        },
+        "expected_date": {
+          "type": "string",
+          "format": "date",
+          "title": "The expected date for the issue",
+          "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
+          "validationMessage": "Should be in the following format: 2022-12-31 (YYYY-MM-DD).",
+          "form": {
+            "type": "datepicker",
+            "wrappers": [
+              "form-field"
+            ],
+            "validation": {
+              "messages": {
+                "pattern": "Should be in the ISO 8601, YYYY-MM-DD."
+              }
+            }
+          }
+        },
+        "regular": {
+          "title": "Indicates if issue is regular or irregular",
+          "description": "If false, no links with the holdings patterns.",
```

```suggestion
          "title": "Regularity",
          "description": "Indicates if issue is regular or not. If false, no links will be added to the holdings patterns.",
```
I'm not sure about my proposition for the description property. Maybe I don't understand exactly what is a regular/irregular issue? Not compliant with the pattern?

---

I removed the `description` because it is a field that added by the system. librarian does not control it for now